### PR TITLE
Add responsiveness

### PR DESCRIPTION
**Pull request by osylvain**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,41 +24,41 @@
...
-                            <div class="register-button-container">
-                                <a id="tweet" class="button tweet-button" href="https://twitter.com/share?url=http%3A%2F%2Finvoked.io&text=Just%20registered%20for%20invoked.io%20private%20beta.%20Embrace%20scaling%20at%20function%20level&via=mentum_io" target="_blank">Tweet <i class="fa fa-twitter"></i> </a>
-                                <span class="or-separator">or</span>
-                                <button id="pay-button" class="button  pay-button">Invest 1$</button>
+                        <div class="col-md-6">
+                            <div class="register-panel">
+                                <h3>Get full access before everyone</h3>
+                                <ul>
+                                    <li>Get access to the private beta</li>
+                                    <li>Get 100$ of free cloud credits on public release</li>
```
Comment (EFF) [inline index.html]: here I would choose between 
- get 100$ credits on public release
- get a larger free tier on public release 

larger free tier looks kinda gay

File: index.html [hunk 2]
```diff
@@ -92,31 +102,36 @@
...
-                        <button class="button no-fill-button select-langage" data-langage="go">GO</button>
-                        <button class="button no-fill-button select-langage" data-langage="cs">C#</button>
-                        <button class="button no-fill-button select-langage" data-langage="ruby">Ruby</button>
-                        <button class="button no-fill-button select-langage" data-langage="python">Python</button>
-                        <button class="button no-fill-button select-langage" data-langage="sh">Curl</button>
+                    <div class="col-sm-5 code-example-right">
+                        <div>
+                            <h2>Invoke any remote function in your favourite language.</h2>
+                            <div class="buttons">
+                                <button class="button no-fill-button select-language active" data-language="js">Nodejs</button>
```
Comment (EFF) [inline index.html:114]: "excuse my french" : http://en.wikipedia.org/wiki/Excuse_My_French_%28album%29

File: style.css [hunk 3]
```diff
@@ -68,12 +66,17 @@ section a {
...
 }
 
-.no-fill-button:hover, .no-fill-button.active{
+.no-fill-button:hover, .no-fill-button.active {
     border-color: #20677B;
     background-color: #20677B;
     color: white;
 }
 
+div [class*=col-md-6] {
```
Comment (EFF) [inline style.css:81]: looks weird to me .. is it for the display flex bug with bootstrap ? 

if yes, could you add a comment on top of it so it clear it up

**Comment by EFF** [comment]
### YES MAN !

File: style.css (hunk 3 above)
Comment (osylvain) [inline style.css:81]: I have no idea what that is for :S. Will check.

**Comment by slvnperron** [comment]
Great job! :shipit:

**Pull request merged**